rundeckapp/grails-app/services/rundeck/services/ExecutionService.groovy

```diff
@@ -795,13 +795,6 @@ class ExecutionService implements ApplicationContextAware, StepExecutor, NodeSte
     def cleanupRunningJobs(String serverUUID = null, String status = null, Date before = new Date()) {
         cleanupRunningJobs(findRunningExecutions(serverUUID, before), status)
     }
-    /**
-     * Set the result status to FAIL for any Executions that are not complete, does not create a new transaction
-     * @param serverUUID if not null, only match executions assigned to the given serverUUID
-     */
-    def cleanupRunningJobs_currentTransaction(String serverUUID = null, String status = null, Date before = new Date()) {
```

Comment:
Note: these functions are removed since updating them to produce the desired behavior (committing `execution` records in a separate inner transaction) would make them functionally equivalent to `def saveExecutionState(schedId, exId, Map props, AsyncStarted execmap, Map retryContext)`